xmtp_api_d14n/src/lib.rs

```diff
@@ -51,6 +51,17 @@ pub mod tests {
     {
         type Builder = D14nClientBuilder<C::Builder, Payer::Builder>;
 
+        fn local_port() -> &'static str {
+            "5055"
```

Comment:
Nit: can we extract this as a config?

Reply:
I think it's fair to keep them here for now since they're specific to this implementation.